README.md

```diff
@@ -5,22 +5,25 @@ Build scripts to help generate cross ToolChains for various platforms supported
 
 ## Dependencies
 
+* git
+* ncurses
 * gperf
 * help2man
 * bison
 * texinfo
 * flex
 * gawk
+* unzip
 
 ## For Arch users:
 ```bash
-sudo pacman -S base-devel gperf help2man bison texinfo flex gawk
+sudo pacman -S base-devel git gperf help2man bison texinfo flex gawk unzip
 ```
 
 ## For Debian/Ubuntu users:
 
 ```bash
-sudo apt-get install build-essential gperf help2man bison texinfo flex gawk autoconf automake wget curl file libncurses-dev
+sudo apt-get install build-essential git gperf help2man bison texinfo flex gawk autoconf automake wget curl file libncurses-dev unzip libtool libtool-bin
```

Comment:
Is there any method to the order? I'd be inclined to put the autoconf/automake/libtool behind build-essential. :-P

Reply:
There was a vague attempt at alphabetical order, and then I went "fuck it" halfway through ;p.